File: support_tools/swigwin-3.0.12/Examples/test-suite/python/inout_runme.py
import inout

a = inout.AddOne1(1)
if a != 2:
    raise RuntimeError

a = inout.AddOne3(1, 1, 1)
if a != [2, 2, 2]:
    raise RuntimeError

a = inout.AddOne1p((1, 1))
if a != (2, 2):
    raise RuntimeError

a = inout.AddOne2p((1, 1), 1)
if a != [(2, 2), 2]:
    raise RuntimeError

a = inout.AddOne3p(1, (1, 1), 1)
if a != [2, (2, 2), 2]:
    raise RuntimeError

File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/inout_runme.m
inout

a = inout.AddOne1(1);
if (a != 2)
  error
endif

a = inout.AddOne3(1,1,1);
if (a != [2,2,2])
  error
endif

a = inout.AddOne1p((1,1));
if (a != (2,2))
  error
endif

a = inout.AddOne2p((1,1),1);
if (a != [(2,2),2])
  error
endif

a = inout.AddOne3p(1,(1,1),1);
if (a != [2,(2,2),2])
  error
endif

